server/heroku/processes.go

```diff
@@ -91,6 +92,15 @@ func (h *PostProcess) ServeHTTPContext(ctx context.Context, w http.ResponseWrite
 
 		fmt.Fprintf(outStream, "HTTP/1.1 200 OK\r\nContent-Type: application/vnd.empire.raw-stream\r\n\r\n")
 
+		// We send the null character periodically, to keep the connection alive.
+		// Otherwise the connection would close after the ELB idle connection timeout.
+		go func() {
+			for {
+				fmt.Fprintf(outStream, "\x00")
```

Comment:
I added a package under pkg/stream/http. Might be nice to throw that heartbeat function in there :).

Maybe it would be better to return a stop channel, instead of passing it in.

```go
stop := streamhttp.Heartbeat(w, 10*time.Second)
defer close(stop)
```